[2024-04-02T03:15:46.585+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag_pythonop.first_task manual__2024-04-02T03:15:42.884531+00:00 [queued]>
[2024-04-02T03:15:46.602+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag_pythonop.first_task manual__2024-04-02T03:15:42.884531+00:00 [queued]>
[2024-04-02T03:15:46.603+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-02T03:15:46.634+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): first_task> on 2024-04-02 03:15:42.884531+00:00
[2024-04-02T03:15:46.643+0000] {standard_task_runner.py:60} INFO - Started process 1168 to run task
[2024-04-02T03:15:46.647+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'first_dag_pythonop', 'first_task', 'manual__2024-04-02T03:15:42.884531+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/first_day_pythonop.py', '--cfg-path', '/tmp/tmpxvwry1pk']
[2024-04-02T03:15:46.649+0000] {standard_task_runner.py:88} INFO - Job 98: Subtask first_task
[2024-04-02T03:15:46.726+0000] {task_command.py:423} INFO - Running <TaskInstance: first_dag_pythonop.first_task manual__2024-04-02T03:15:42.884531+00:00 [running]> on host 5c9eefa8eb82
[2024-04-02T03:15:46.879+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_dag_pythonop' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-02T03:15:42.884531+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-02T03:15:42.884531+00:00'
[2024-04-02T03:15:46.882+0000] {python.py:202} INFO - Done. Returned value was: hello world
[2024-04-02T03:15:46.922+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=first_dag_pythonop, task_id=first_task, execution_date=20240402T031542, start_date=20240402T031546, end_date=20240402T031546
[2024-04-02T03:15:46.982+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-02T03:15:47.012+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
